next-app/src/app/(app-layout)/flavours/[flavourId]/page.tsx

```diff
@@ -3,69 +3,71 @@ import { PageTitle } from '@/common/page-title';
 import { Paper } from '@/common/paper';
 import { Section, SectionTitle } from '@/common/section';
 import { getMetadata } from '@/seo/seo-utils';
-import { RelatedProducts } from '@/whiskeys/related-whiskey';
-import { WhiskeyDetails } from '@/whiskeys/whiskey-details';
-import { WhiskeyGridSkeleton } from '@/whiskeys/whiskey-grid';
-import type { Metadata } from 'next';
-import { notFound } from 'next/navigation';
-import { Suspense } from 'react';
-import { getObjectById } from '@/db/db-utils';
 import { Flavour } from '@/common/object-types';
+import { getObjectById } from '@/db/db-utils';
+import { Metadata } from 'next';
+import Image from 'next/image';
+import { notFound } from 'next/navigation';
 
-
-export type ProductPageProps = {
+type FlavourPageProps = {
   params: {
-    flavourId: Id;
+    flavourId: Id | string | number;
```

Comment:
this can be just type "Id", no need for other types. To ensure it is Id, assert it, and throw a 404 if not of type

Reply:
Right, let me fix it